cachedb-load-data/job-load-cachedb-data.yaml

```diff
@@ -0,0 +1,51 @@
+﻿apiVersion: batch/v1
+kind: Job
+metadata:
+  name: load-cachedb-data
+  labels:
+    job-name: load-cachedb-data
+spec:
+  parallelism: 1
+  completions: 1
+  backoffLimit: 6
+  template:
+    metadata:
+      name: load-cachedb-data
+      labels:
+        job-name: load-cachedb-data
+    spec:
+      containers:
+        - name: cachedb-loader
+          # TODO the rest of the charts users /rsotf/ namespace, but this one uses /retail-infra/ namespace
+          image: image-registry.openshift-image-registry.svc:5000/retail-infra/cachedb-loader
+          resources: {}
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          imagePullPolicy: Always
+          env:
+            - name: POSTGRES_HST
+              value: postgres-svc
+#            - name: POSTGRES_PRT
+#              value: value: {{ include "postgres.servicePort" . | toString | quote }}
```

Comment:
What about this? We should add some comment if this is needed/should be left

Reply:
Added an explicit TODO statement for now (so we could improve the functionality by enabling port definition).